src/Microsoft.Diagnostics.EventFlow.ServiceFabric/ServiceFabricDiagnosticPipelineFactory.cs

```diff
@@ -14,6 +17,9 @@ namespace Microsoft.Diagnostics.EventFlow.ServiceFabric
 {
     public static class ServiceFabricDiagnosticPipelineFactory
     {
+        public static readonly string ConfigurationPackageName = "Config";
```

Comment:
nit: Is this intended to be the same as ServiceFabricConfigurationProvider.DefaultConfigurationPackageName? I can't determine that buy reading the code. If it is, I think it would be good to reuse the constant just in case it changes for some unforeseen reason. 
That said, not very worried. I don't have a problem keep it as is either.